api/graphql/tests/test_reservations/test_reservation_approve.py

```diff
@@ -70,8 +70,8 @@ def get_valid_approve_data(self):
         return {
             "pk": self.reservation.pk,
             "handlingDetails": "You're welcome.",
-            "price": 10.59,  # This floating point number will float somewhere 10.58999...
-            "priceNet": 8.61,
+            "price": "10.59",  # This floating point number will float somewhere 10.58999...
```

Comment:
I'm assuming this comment is not valid anymore ☺️

Reply:
cleaned 🧹